pulp-api/sign-metadata.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+FILE_PATH=$1
+SIGNATURE_PATH="$1.asc"
+ADMIN_ID=$(gpg --list-keys --with-colons --homedir ~/.gnupg/ | grep fpr: | head -n1 | cut -d: -f10)
```

Comment:
I'd really much rather see a full path for the homedir than "current user's $HOME/.gnupg".  That could be set at the top of the script using `export GNUPGHOME=...` and the parameter removed from both commands, increasing the "don't repeat yourself"-ness.